web_src/less/themes/theme-arc-green.less

```
@@ -1109,6 +1109,9 @@ input {
 .lines-code .hljs {
     background-color: #2a2e3a !important;
 }
+.blob-excerpt {
```

To override the `gitea` theme
```suggestion
td.blob-excerpt {
```

---

it was als intended to leave td because it is more flexible when template changes ocour

---

should I leave it or add the `td` ? but then it should also be added to `.lines-code`

---

If you take a look at #10181 it only needs to be added to this one.   
I would prefer to avoid an `!important` if we can.